### PR TITLE
W-16420208 Tweaks to mule-gateway-config-autodiscovery-mule4.adoc

### PR DESCRIPTION
**Pull request by dukesphere**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: mule-gateway/modules/ROOT/pages/mule-gateway-config-autodiscovery-mule4.adoc [hunk 1]
```diff
@@ -32,12 +32,12 @@ This step is necessary to instruct API Manager to use your deployed Mule applica
 
 == Configuring Anypoint Platform Organization Credentials in Mule Runtime
 
-To use Autodiscovery in a Mule Application, the Runtime must start with Anypoint Platform credentials already configured. +
+To use Autodiscovery in a Mule application, Mule runtime engine must start with Anypoint Platform credentials already configured.
```
Comment (dukesphere) [inline mule-gateway/modules/ROOT/pages/mule-gateway-config-autodiscovery-mule4.adoc:35]: See "Mule application" and "Mule runtime engine (Mule or Mule runtime)" in the term ref: https://confluence.internal.salesforce.com/display/MTDT/Term+Presentation+Reference#TermPresentationReference-M

Comment (dukesphere) [reply mule-gateway/modules/ROOT/pages/mule-gateway-config-autodiscovery-mule4.adoc:35]: I believe we discourage the use of "+" in regular paragraphs like this due to the non-standard formatting results.

File: mule-gateway/modules/ROOT/pages/mule-gateway-config-autodiscovery-mule4.adoc [hunk 2]
```diff
@@ -32,12 +32,12 @@ This step is necessary to instruct API Manager to use your deployed Mule applica
...
 == Configuring Anypoint Platform Organization Credentials in Mule Runtime
 
-To use Autodiscovery in a Mule Application, the Runtime must start with Anypoint Platform credentials already configured. +
+To use Autodiscovery in a Mule application, Mule runtime engine must start with Anypoint Platform credentials already configured. 
 Learn how to configure your organization credentials based on your deployment target in xref:api-manager::org-credentials-config-mule4.adoc[Configuring Organization Credentials in Mule Runtime 4].
 
 == Configuring the Autodiscovery Element in Your Mule Application
 
-Within the code of your Mule Application, you must configure the `api-gateway:Autodiscovery` element. +
+Within the code of your Mule application, you must configure the `api-gateway:Autodiscovery` element.
```
Comment (dukesphere) [inline mule-gateway/modules/ROOT/pages/mule-gateway-config-autodiscovery-mule4.adoc:40]:
```suggestion
Within the code of your Mule application, you must configure the `api-gateway:Autodiscovery` element. 

```

File: mule-gateway/modules/ROOT/pages/mule-gateway-config-autodiscovery-mule4.adoc (hunk 1 above)
Comment (dukesphere) [reply mule-gateway/modules/ROOT/pages/mule-gateway-config-autodiscovery-mule4.adoc:35]:
```suggestion
To use Autodiscovery in a Mule application, Mule runtime engine must start with Anypoint Platform credentials already configured. 

```

alternative on a separate line

**Review by marinasasso — APPROVED** [review]

**Pull request merged**